rust/processor/src/processors/token_v2_processor.rs

```diff
@@ -94,172 +87,83 @@ async fn insert_to_db(
         "Inserting to db",
     );
 
-    let coll_v2 = execute_in_chunks(
-        conn.clone(),
-        insert_collections_v2_query,
-        collections_v2,
-        get_config_table_chunk_size::<CollectionV2>("collections_v2", per_table_chunk_sizes),
-    );
-    let td_v2 = execute_in_chunks(
-        conn.clone(),
-        insert_token_datas_v2_query,
-        token_datas_v2,
-        get_config_table_chunk_size::<TokenDataV2>("token_datas_v2", per_table_chunk_sizes),
+    let coll_v2 = db_writer.send_in_chunks("collections_v2", collections_v2);
+    let td_v2 = db_writer.send_in_chunks("token_datas_v2", token_datas_v2);
+    let to_v2 = db_writer.send_in_chunks("token_ownerships_v2", token_ownerships_v2);
+    let cc_v2 = db_writer.send_in_chunks("current_collections_v2", current_collections_v2);
+    let ctd_v2 = db_writer.send_in_chunks("current_token_datas_v2", current_token_datas_v2);
+    let cto_v2 =
+        db_writer.send_in_chunks("current_token_ownerships_v2", current_token_ownerships_v2);
+    let cbto_v2 = db_writer.send_in_chunks(
+        "current_token_ownerships_v2",
+        current_burned_token_ownerships_v2,
     );
```

Comment:
currently the model associates a `Vec<Item>` with a single query. i feel like this is the wrong model because there can be multiple queries that can be run on a Vec of the same items. I think better UX would be at `db_writer.send` time, decide which vec + query combo to send to db writer. maybe have the execute_query fn as part of `QueryGenerator`? not sure how feasible this would be though.

Reply:
Fixed :-)